src/submission/migration/migration-manager.ts

```diff
@@ -134,10 +134,15 @@ export namespace MigrationManager {
 
     const newSchemaVersion = migrationToRun.toVersion;
     const dictionaryName = dictionaryManagerInstance().getCurrentName();
-    const newTargetSchema = await dictionaryManagerInstance().loadSchemaByVersion(
-      dictionaryName,
-      newSchemaVersion,
-    );
+    let newTargetSchema: dictionaryEntities.SchemasDictionary;
+    try {
+      newTargetSchema = await dictionaryManagerInstance().loadSchemaByVersion(
+        dictionaryName,
+        newSchemaVersion,
+      );
+    } catch (err) {
+      return await abortMigration(migrationToRun);
```

Comment:
so `abortMigration` takes a 2nd argument where it stores the failure analysis right, could you modify it so it also accepts a message like `schema loading failed, check that version is correct or try again`

Reply:
or any other way to indicate the reason of abortion

Reply:
![Screen Shot 2020-07-24 at 9 43 41 AM](https://user-images.githubusercontent.com/8744047/88397617-349f5d00-cd92-11ea-8fcf-539dc1064f68.png)